src/core/friendly_errors/sketch_verifier.js

```diff
@@ -1,11 +1,64 @@
 import * as acorn from 'acorn';
 import * as walk from 'acorn-walk';
+import * as constants from '../constants';
 
 /**
  * @for p5
  * @requires core
  */
 function sketchVerifier(p5, fn) {
+  // List of functions to ignore as they either are meant to be re-defined or
+  // generate false positive outputs.
+  const ignoreFunction = [
+    'setup',
+    'draw',
+    'preload',
+    'deviceMoved',
+    'deviceTurned',
+    'deviceShaken',
+    'doubleClicked',
+    'mousePressed',
+    'mouseReleased',
+    'mouseMoved',
+    'mouseDragged',
+    'mouseClicked',
+    'mouseWheel',
+    'touchStarted',
+    'touchMoved',
+    'touchEnded',
+    'keyPressed',
+    'keyReleased',
+    'keyTyped',
+    'windowResized',
+    'name',
+    'parent',
+    'toString',
+    'print',
+    'stop',
+    'onended'
```

Comment:
The functions meant to be redefined by the user make sense. Out of curiosity where do the other false positive entries come from? Is that from looking at `classesWithGlobalFunctions`?